<ui version="4.0" >
 <class>FiberVisClass</class>
 <widget class="QWidget" name="FiberVisClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>FiberVis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>2</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>3</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="controlFrame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <property name="horizontalSpacing" >
       <number>3</number>
      </property>
      <property name="verticalSpacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item row="0" column="4" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" >
       <widget class="QGroupBox" name="volumeGroup" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Show tensor field</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" >
         <property name="horizontalSpacing" >
          <number>5</number>
         </property>
         <property name="verticalSpacing" >
          <number>4</number>
         </property>
         <property name="margin" >
          <number>5</number>
         </property>
         <item row="0" column="2" >
          <widget class="QSlider" name="axialSlider" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition" >
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval" >
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QCheckBox" name="axialCheck" >
           <property name="text" >
            <string>Axial</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QCheckBox" name="coronalCheck" >
           <property name="text" >
            <string>Coronal</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QCheckBox" name="sagittalCheck" >
           <property name="text" >
            <string>Sagittal</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QSlider" name="coronalSlider" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2" >
          <widget class="QSlider" name="sagittalSlider" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item rowspan="3" row="0" column="3" >
          <widget class="Line" name="line" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="4" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Display</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5" >
          <widget class="QComboBox" name="dispCombo" >
           <item>
            <property name="text" >
             <string>B0</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Color</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>FA</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Tractography</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Color Mode</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="colorCombo" >
           <item>
            <property name="text" >
             <string>Local Orientation</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Cluster</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Seed Region</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Thickness</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QDoubleSpinBox" name="thickSpin" >
           <property name="minimum" >
            <double>0.100000000000000</double>
           </property>
           <property name="maximum" >
            <double>5.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>0.100000000000000</double>
           </property>
           <property name="value" >
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Viewpoint</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="comboViewpoint" >
           <item>
            <property name="text" >
             <string>Free</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Top</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Frontal</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Left</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QGroupBox" name="groupBox_3" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Clusters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Cluster file:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="lineClusterFile" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QPushButton" name="btnClusterFile" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>Display cluster:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2" >
          <widget class="QFrame" name="frame" >
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <property name="spacing" >
             <number>1</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnPrevCluster" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>&lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineCluster" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>-1</string>
              </property>
              <property name="readOnly" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelNumClusters" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string> / 0 </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnNextCluster" >
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnAllClusters" >
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>All</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0" colspan="3" >
          <widget class="QPushButton" name="btnTest" >
           <property name="text" >
            <string>Save selected index</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QGroupBox" name="groupBox_4" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Embedding</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6" >
         <property name="horizontalSpacing" >
          <number>3</number>
         </property>
         <property name="verticalSpacing" >
          <number>5</number>
         </property>
         <property name="margin" >
          <number>3</number>
         </property>
         <item row="2" column="0" >
          <widget class="QCheckBox" name="checkShowEmbed" >
           <property name="text" >
            <string>Show embeddings</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QCheckBox" name="checkShowLabel" >
           <property name="text" >
            <string>Show cluster labels</string>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0" >
          <widget class="QCheckBox" name="checkColorByTpl" >
           <property name="text" >
            <string>Color by template cluster</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2" >
          <widget class="QFrame" name="frame_3" >
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <property name="spacing" >
             <number>3</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>Embedding file:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEmbedFile" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnEmbedFile" >
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QPushButton" name="btnAmb" >
           <property name="text" >
            <string>Show ambigous embs</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QPushButton" name="btnAddAtlas" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Add to atlas</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" >
          <widget class="QDoubleSpinBox" name="spinMinDist" >
           <property name="value" >
            <double>30.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="dispFrame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>axialCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>axialSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>57</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>coronalCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>coronalSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>61</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sagittalCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>sagittalSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>65</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAmb</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnAddAtlas</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>970</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel" >
     <x>970</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
